lib/killrvideo-docker-common/docker-compose-external.yaml

```diff
@@ -54,20 +57,28 @@ services:
       KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME
       KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD
       NODE_ENV: $NODE_ENV
+      KILLRVIDEO_ENABLE_SSL: $KILLRVIDEO_ENABLE_SSL
       KILLRVIDEO_CASSANDRA_REPLICATION: $KILLRVIDEO_CASSANDRA_REPLICATION
       KILLRVIDEO_LOGGING_LEVEL: $KILLRVIDEO_LOGGING_LEVEL
+      KILLRVIDEO_YOUTUBE_API_KEY: $KILLRVIDEO_YOUTUBE_API_KEY
```

Comment:
What will KILLRVIDEO_YOUTUBE_API_KEY be used for?

Reply:
The youtube key is used to work with their back-end API.  This is how we grab video content from YouTube and display it on KillrVideo.

Reply:
You can see this in action if you take a look at the generator service.